audio-worklet/src/bitcrusher-controller.js

```diff
@@ -30,112 +30,112 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+    
+    this.gainNode_ = new GainNode(this.context);
+    this.gainNode_.gain.value = gain || 0.5;
+  }
 
-	/**
-	 * Initalize HTML elements when document has loaded
-	 * @param  {String} containerId the id of parent container
-	 */
-	initializeGUI(containerId) {
-		this.sourceButton_ = new SourceController(containerId, this.start.bind(this), 
-			this.stop.bind(this));
-		this.sourceButton_.disable();
-		
-		// Place 3 parameters in container to real-time adjust Bitcrusher_ settings
-		this.bitDepthParam_ = new ParamController(containerId, 
-			this.setBitDepth.bind(this), {type: "range", min: 1, max: 24,
-			step: 0.1, default: 24, name: "Bit Depth"});
-		this.bitDepthParam_.disable();
+  /**
+   * Initalize HTML elements when document has loaded
+   * @param  {String} containerId the id of parent container
+   */
+  initializeGUI(containerId) {
+    this.sourceButton_ = new SourceController(containerId, this.start.bind(this), 
+      this.stop.bind(this));
+    this.sourceButton_.disable();
+    
+    // Place 3 parameters in container to real-time adjust Bitcrusher_ settings
+    this.bitDepthParam_ = new ParamController(containerId, 
+      this.setBitDepth.bind(this), {type: "range", min: 1, max: 24,
+      step: 0.1, default: 24, name: "Bit Depth"});
+    this.bitDepthParam_.disable();
 
-		this.reductionParam_ = new ParamController(containerId, 
-			this.setReduction.bind(this), {type: "range", min: 1, max: 20,
-			step: 0.1, default: 1, name: "Sample Rate Reduction"});
-		this.reductionParam_.disable();
+    this.reductionParam_ = new ParamController(containerId, 
+      this.setReduction.bind(this), {type: "range", min: 1, max: 20,
+      step: 0.1, default: 1, name: "Sample Rate Reduction"});
+    this.reductionParam_.disable();
 
-		this.gainParam = new ParamController(containerId, 
-			this.setGain.bind(this), {type: "range", min: 0, max: 1,
-			step: 0.01, default: 0.5, name: "Volume"});
+    this.gainParam = new ParamController(containerId, 
+      this.setGain.bind(this), {type: "range", min: 0, max: 1,
+      step: 0.01, default: 0.5, name: "Volume"});
 
-		const songUrl = "audio/it-from-bit.mp3";
-		let that = this;
+    const songUrl = "audio/it-from-bit.mp3";
+    let that = this;
 
-		// Fetch song from server and decode into buffer
-		fetch(songUrl).then((response) => { 
-			return response.arrayBuffer();})
-				.then((song) => { 
-					that.context.decodeAudioData(song)
-						.then((buffer) => {
-		  				that.songBuffer = buffer;
-		  				that.sourceButton_.enable();
-		  			}, (message) => {console.error(message)});
-		});
-	}
+    // Fetch song from server and decode into buffer
+    fetch(songUrl).then((response) => { 
+      return response.arrayBuffer();})
+        .then((song) => { 
+          that.context.decodeAudioData(song)
+            .then((buffer) => {
+              that.songBuffer = buffer;
+              that.sourceButton_.enable();
+            }, (message) => {console.error(message)});
```

Comment:
The rejected promise isn't actually called with a "message".  It's the error type, so you should probably pick a better variable name. And if you want to handle this, you should print out a little more than logging the message to the console.  We try to be very verbose on error conditions.  (See for example https://bugzilla.mozilla.org/show_bug.cgi?id=996699)